torchci/components/tritonbench/common.tsx

```diff
@@ -0,0 +1,18 @@
+export const DEFAULT_TRITON_REPOSITORY = "triton-lang/triton";
+export const DEFAULT_TRITON_BENCHMARK_NAME = "nightly";
+export const DEFAULT_DEVICE_NAME = "NVIDIA H100";
+
+export const BENCHMARK_NAME_METRICS_MAPPING: { [key: string]: any } = {
+  nightly: ["tflops-avg"],
```

Comment:
are all the op benchmarks using consistent naming if not should it be enforced in triton bench, e.g. is it depdent on this name: https://github.com/pytorch-labs/tritonbench/blob/15eb47c19f56ebc3396f72ddda89d50a77a0264f/tritonbench/operators/flex_attention/operator.py#L266-L267 ?

Reply:
Tritonbench forces the naming metric of `tflops` across all the op benchmarks.